.env.sample

```diff
@@ -4,5 +4,7 @@ SCHEME=http
 ROCKET_SECRET_KEY=Qt6DPFUU8qO4BKTCQnKAgt9FBBJxIWAYUGyHuruVfpE=
 ROCKET_LOG=normal
 ROCKET_DATABASES={service_cache={url="redis://127.0.0.1:6379"}}
+ROCKET_PORT=8000
+ROCKET_ADDRESS=localhost
```

Comment:
Does this break docker compose if you 1:1 copy it to .env? (In the docker file we build the prod version and there the default address is `0.0.0.0` and not localhost. If this is overwritten it might be that the service is not reachable ... not sure how the port forwarding works)

Reply:
I am totally fine with 0.0.0.0 if that solves it. I thought the default is localhost. Let me check if it is a problem.

Reply:
It depends on your environment. Docker is configured to use the release env and local uses the dev environment, both have different default (as development should not be reachable externally)

Reply:
When running the service with `docker-compose up` it is not reachable via telnet either way. Having `0.0.0.0` or `localhost` in .env don't work. The connection is established and dropped immediately. Usually it is necessary to expose ports with docker. But I don't know how that works with `docker-compose up`.

Reply:
The question was more if the web server is accessible either way.